job_templates/sag_pt_ddp/app_1/config_fed_client.conf

```diff
@@ -46,18 +46,14 @@
           # difference VS received parameters and send the difference
           params_transfer_type = "DIFF"
 
-          # if training is true, the executor will expect
-          # the custom code need to send back the trained parameters
-          training = true
-
-          # if global_evaluation is true, the executor will expect the
-          # custom code to send back the evaluation metric
-          global_evaluation = true
+          # if train_with_evaluation is true, the executor will expect
+          # the custom code need to send back both the trained parameters and the evaluation metric
```

Comment:
"otherwise only trained parameters are expected"?

Reply:
ok let me fix this